acra/regulations/_index.md

```diff
@@ -2,3 +2,50 @@
 title: Regulations
 bookCollapseSection: true
 ---
+
+# Acra and GDPR compliance
+
+## How Acra can help you comply with GDPR
+
+Acra is an encryption tool that allows processing sensitive data with great care and secureness — namely, encrypting the data in a certain way. Acra performs encryption through [Themis]({{< ref "/themis/" >}}) in [Secure Cell]({{< ref "themis/crypto-theory/cryptosystems/secure-cell.md" >}})/Seal mode (AES256 + GCM). This corresponds to the &quot;state of the art&quot; (as mentioned in the [full text of the GDPR](https://gdpr-info.eu/) regulation) algorithms of symmetric encryption. Also, Acra is securely pre-configured by default and carries out extensive logging of all its actions &quot;out of the box&quot;. Using Acra helps reaching compliance with some of the demands of the articles 32 ([Security of processing](https://gdpr-info.eu/art-32-gdpr/)) and 25 ([Data protection by design and by default](https://gdpr-info.eu/art-25-gdpr/)) of GDPR, as well as helps comply with the articles 33 and 34  ([Notification of a personal data breach to the supervisory authority](http://gdpr-info.eu/art-33-gdpr/) and [Communication of a personal data breach to the data subject](http://gdpr-info.eu/art-34-gdpr/)) of GDPR.
+
+## Security of processing
+[Article 32 of GDPR](https://gdpr-info.eu/art-32-gdpr/)
+
+Acra provides &quot;state of the art security of processing&quot; required in article 32 of GDPR through providing data encryption and integrity check of the encrypted data.
+
+_&gt; &quot;Taking into account the state of the art, the costs of implementation and the nature, scope, context and purposes of processing as well as the risk of varying likelihood and severity for the rights and freedoms of natural persons, the controller and the processor shall implement appropriate technical and organisational measures to ensure a level of security appropriate to the risk,...&quot;_
```

Comment:
And I believe this quote should be moved closer to the link to GDPR so that it reads more naturally:

```markdown
## Security of processing

[Article 32 of GDPR](https://gdpr-info.eu/art-32-gdpr/):

> Taking into account the state of the art [...]

Acra provides “state of the art security of processing” [...]
```